apps/dashboard/app/models/project.rb

```diff
@@ -3,6 +3,7 @@
 class Project
   include ActiveModel::Model
   include ActiveModel::Validations
+  include ActiveModel::Validations::Callbacks
```

Comment:
This was from a previous strategy that was backed out and now this needs removed.